server/storage-access-layer/pom.xml

```diff
@@ -49,6 +64,20 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
this version can also managed by parent pom at "pluginManagement"

Reply:
Done